src/exchange/actions.rs

```diff
@@ -101,6 +101,14 @@ pub struct BulkOrder {
     pub grouping: String,
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct BulkOrderWithBuilder {
```

Comment:
This doesn't have to be an extra struct. Can reduce copy/paste here and reusability by adding
```
#[serde(default, skip_serializing_if = "Option::is_none")]
pub builder: BuilderInfo,
```
to `BulkOrder`